### PR TITLE
protobuf: refresh the python resources

### PR DESCRIPTION
**Pull request by ilovezfs**

### CODE REVIEW

File: Formula/protobuf.rb [hunk 1]
```diff
@@ -51,23 +51,24 @@ class Protobuf < Formula
...
+    url "https://pypi.python.org/packages/source/s/setuptools/setuptools-20.9.0.tar.gz"
+    sha256 "2a360c782e067f84840315bcdcb5ed6c7c841cdedf6444f3232ab4a8b3204ac1"
   end
 
   resource "six" do
-    url "https://pypi.python.org/packages/source/s/six/six-1.9.0.tar.gz"
-    sha256 "e24052411fc4fbd1f672635537c3fc2330d9481b18c0317695b46259512c91d5"
+    url "https://pypi.python.org/packages/source/s/six/six-1.10.0.tar.gz"
+    sha256 "e24052411fc4fbd1f672635537c3fc2330d9481b18c0317695b46259512c91d4"
+    sha256 "105f8d68616f8248e24bf0e9372ef04d3cc10104f1980f54d57b2ce73a5ad56a"
```
Comment (xu-cheng) [inline Formula/protobuf.rb]: Two sha256?

Comment (ilovezfs) [inline Formula/protobuf.rb]: fixed.

**Comment by ilovezfs** [comment]
OK, after doing a bunch of updates in homebrew science, it looks like CI now passes for this.

**Comment by mikemcquaid** [comment]
👍 :shipit:

**Pull request closed without merging**